hack/auth-config-template.yaml

```diff
@@ -0,0 +1,12 @@
+# see Region Identifier in https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm
+# for a list of regions
+region: <region>
+tenancy: <tenancy OCID>
+user: <user OCID>
+key: |
+  -----BEGIN RSA PRIVATE KEY-----
+  <key info> 
+  -----END RSA PRIVATE KEY-----
+fingerprint: <fingerprint>
```

Comment:
I believe there are actual example values one can use for these instead of using placeholders. Ping me and I'll point you at internal docs that provide the values or I won't be able to find them, in which case you can ignore this review comment completely. :)